Sources/Rendering/OpenGL/Texture/index.js

```diff
@@ -1226,18 +1231,34 @@ function vtkOpenGLTexture(publicAPI, model) {
     model.volumeInfo.dataComputedScale = computedScale;
     model.volumeInfo.dataComputedOffset = computedOffset;
 
-    const useHalfFloat = checkUseHalfFloat(
-      dataType,
-      computedOffset,
-      computedScale,
-      preferSizeOverAccuracy
-    );
+    // Check whether 16bit texture extension is available
+    model.useNorm16 =
+      useExperimentalNorm16Texture &&
```

Comment:
Can we drop `useExperimentalNorm16Texture` and just use the extension presence as the toggle condition? It may be useful to only cache this runtime `useNorm16` flag once as well.